pandas/_libs/missing.pyx

```diff
@@ -10,10 +10,13 @@ cnp.import_array()
 
 cimport pandas._libs.util as util
 
-from pandas._libs.tslibs.np_datetime cimport (
-    get_timedelta64_value, get_datetime64_value)
+
+from pandas._libs.tslibs.np_datetime cimport get_datetime64_value, get_timedelta64_value
 from pandas._libs.tslibs.nattype cimport (
-    checknull_with_nat, c_NaT as NaT, is_null_datetimelike)
+    c_NaT as NaT,
+    checknull_with_nat,
+    is_null_datetimelike,
+)
```

Comment:
maybe wait for isort 5.0.0 (or try out the dev version) to save the manual effort... https://github.com/timothycrosley/isort/issues/252#issuecomment-570893462

Reply:
Oh good to know, thank you @simonjayhawkins!

Reply:
One concern I have and that is the use case of ```cpn.import_array()```, I am worried that ```isort``` will break some of the modules.

Reply:
yah, we have some patterns in the cython imports that would be tricky to specify in the cfg